scripts/zones/Waughroon_Shrine/npcs/Armoury_Crate.lua

```diff
@@ -404,6 +404,65 @@ local loot =
         },
     },
 
+    -- BCNM Royal Jelly
+    [77] =
+    {
+        {
+            { itemid = 637, droprate = 1000 }, -- Slime oil
```

Comment:
Can you add all these items to items.lua

Reply:
done